### PR TITLE
[FLINK-34707][tests] Update base version for japicmp check

### PR DESCRIPTION
**Pull request by lincoln-lil**
This is japicmp configuration update for master branch

### CODE REVIEW

**Comment by lincoln-lil** [comment]
@JingGe @Myasuka @MartijnVisser  Can someone of you help review this?

**Comment by lincoln-lil** [comment]
@JingGe @Myasuka @MartijnVisser Can someone of you help review this?

File: pom.xml [hunk 1]
```diff
@@ -2358,12 +2358,6 @@ under the License.
 								<exclude>@org.apache.flink.annotation.PublicEvolving</exclude>
 								<exclude>@org.apache.flink.annotation.Internal</exclude>
 								<!-- MARKER: start exclusions; these will be wiped by tools/releasing/update_japicmp_configuration.sh -->
```
Comment (Myasuka) [inline pom.xml:2360]: If we could drop the exclusions, do we still need the markers here?

**Review by Myasuka — COMMENTED** [review]
Thanks for creating this PR, PTAL.

File: pom.xml (hunk 1 above)
Comment (lincoln-lil) [reply pom.xml:2360]: This  marker is used for location by the script, see: https://github.com/apache/flink/blob/release-1.19/tools/releasing/update_japicmp_configuration.sh#L63

**Comment by lincoln-lil** [comment]
@Myasuka Thanks for reviewing! Please help review the cherry-pick one: https://github.com/apache/flink/pull/24514

**Review by Myasuka — APPROVED** [review]
Thanks for the update. LGTM

**Pull request merged**